hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/aggregate/JsonSqlAggregateTest.java

```diff
@@ -185,4 +229,74 @@ private static String createTable() {
         );
         return name;
     }
+
+    private static String createTable2() {
+        String name = randomName();
+        TestBatchSqlConnector.create(
+                sqlService,
+                name,
+                asList("name", "k", "v"),
+                asList(VARCHAR, VARCHAR, VARCHAR),
+                asList(new String[]{"Alice", "department", "dep1"},
+                        new String[]{"Bob", "department", "dep2"},
+                        new String[]{"Alice", "job", "job1"},
+                        new String[]{null, "department", "dep2"},
+                        new String[]{"Bob", "job", "job2"},
+                        new String[]{null, "job", "job1"},
+                        new String[]{null, null, "desc2"},
+                        new String[]{"Bob", "description", null},
+                        new String[]{"Alice", "description", "desc1"})
+        );
+        return name;
+    }
+
+    private void assertRowsWithJsonValues(String sql, Collection<Row> rows) {
```

Comment:
Generally, this method looks overcomplicated and it is not universal. E.g., I tried to play with `JSON_OBJECTAGG`, and wrote this test:

```
        @Test
        public void test_sandbox() {
            String name = "map";
            createMapping(name, String.class, int.class);
            IMap<String, Integer> map = instance().getMap(name);
            map.put("A", 1);
            map.put("B", 1);

            assertRowsWithJsonValues(
                    "SELECT JSON_OBJECTAGG(KEY __key VALUE this) FROM " + name,
                    asList(
                            new Row("{\"A\":1,\"B\":1}")
                    )
            );
        }
```
, but I have no success due to assertion errors below. 
Would you like to refactor it and make it more general? Then you can move it to `SqlJsonTestSupport`. WDYT?